[2021-07-09 06:06:13,065] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bluepi_dag.PostgresToDataLake.extract_user_log_data 2021-07-09T05:00:00+00:00 [queued]>
[2021-07-09 06:06:13,080] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bluepi_dag.PostgresToDataLake.extract_user_log_data 2021-07-09T05:00:00+00:00 [queued]>
[2021-07-09 06:06:13,080] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-09 06:06:13,081] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-07-09 06:06:13,081] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-09 06:06:13,099] {taskinstance.py:1107} INFO - Executing <Task(PostgresToGCSOperator): PostgresToDataLake.extract_user_log_data> on 2021-07-09T05:00:00+00:00
[2021-07-09 06:06:13,104] {standard_task_runner.py:52} INFO - Started process 14955 to run task
[2021-07-09 06:06:13,110] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'bluepi_dag', 'PostgresToDataLake.extract_user_log_data', '2021-07-09T05:00:00+00:00', '--job-id', '342', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/bluepi_dag.py', '--cfg-path', '/tmp/tmpjwvl5syp', '--error-file', '/tmp/tmpwbninfp2']
[2021-07-09 06:06:13,117] {standard_task_runner.py:77} INFO - Job 342: Subtask PostgresToDataLake.extract_user_log_data
[2021-07-09 06:06:13,171] {logging_mixin.py:104} INFO - Running <TaskInstance: bluepi_dag.PostgresToDataLake.extract_user_log_data 2021-07-09T05:00:00+00:00 [running]> on host 3cf0296804bd
[2021-07-09 06:06:13,234] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=natchanon
AIRFLOW_CTX_DAG_ID=bluepi_dag
AIRFLOW_CTX_TASK_ID=PostgresToDataLake.extract_user_log_data
AIRFLOW_CTX_EXECUTION_DATE=2021-07-09T05:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-07-09T05:00:00+00:00
[2021-07-09 06:06:13,237] {sql_to_gcs.py:150} INFO - Executing query
[2021-07-09 06:06:13,248] {base.py:78} INFO - Using connection to: id: my_postgres_conn. Host: 35.247.174.171, Port: 5432, Schema: postgres, Login: exam, Password: ***, extra: {}
[2021-07-09 06:06:14,755] {sql_to_gcs.py:153} INFO - Writing local data files
[2021-07-09 06:06:14,756] {sql_to_gcs.py:202} INFO - Current file count: 1
[2021-07-09 06:06:14,757] {sql_to_gcs.py:164} INFO - Uploading 1 files to GCS.
[2021-07-09 06:06:15,015] {gcs.py:485} INFO - File /tmp/tmpkasjofzc uploaded to raw_data/user_log in bluepi-landing-bucket bucket
[2021-07-09 06:06:15,016] {sql_to_gcs.py:167} INFO - Removing local files
[2021-07-09 06:06:15,040] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=bluepi_dag, task_id=PostgresToDataLake.extract_user_log_data, execution_date=20210709T050000, start_date=20210709T060613, end_date=20210709T060615
[2021-07-09 06:06:15,090] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-09 06:06:15,130] {local_task_job.py:151} INFO - Task exited with return code 0
